src/components/site/SubpageTabs.md

```diff
@@ -0,0 +1,39 @@
+### Subpage Tabs
+
+```jsx
+import SubPageTabs from "./SubPageTabs";
+import SubPageTabItem from "./SubPageTabItem";
+import Section from "../basic/containers/Section";
+
+<Section>
+  <SubPageTabs text="In This Section" id="dg_menu-trigger-btn">
+    <SubPageTabItem className="active" text="Newsroom" link="/home" />
+    <SubPageTabItem text="Social Media Gallery" link="/home" />
+    <SubPageTabItem text="Media Contact Information" link="/home" />
```

Comment:
you probably should make some fake links here so the reflect something similar to the text

Reply:
Updated and committed.